crates/ruma-federation-api/src/transactions/edu.rs

```diff
@@ -227,13 +228,13 @@ pub struct DeviceListUpdateContent {
 }
 
 impl DeviceListUpdateContent {
-    /// Creates a new `DeviceListUpdateContent` with default `prev_id`, `deleted`
-    /// and `keys` fields.
-    pub fn new(user_id: UserId, device_id: DeviceIdBox, name: String, stream_id: UInt) -> Self {
+    /// Creates a new `DeviceListUpdateContent` with default `prev_id`, `deleted`,
+    /// `device_display_name` and `keys` fields.
+    pub fn new(user_id: UserId, device_id: DeviceIdBox, stream_id: UInt) -> Self {
```

Comment:
I wasn't sure about removing the display name from this function.

Reply:
Seems reasonable.